maquetado/index.html

```diff
@@ -0,0 +1,32 @@
+<!doctype html>
+
+<html lang="en">
+
+<head>
+  <meta charset="utf-8">
+  <title>HTML & CSS Training</title>
+  <meta name="description" content="training">
+  <meta name="author" content="Wolox">
+  <link rel="stylesheet" type="text/css" href="application.css">
+</head>
+
+<body>
+  <div class="textAndDescription">
+    <p class="title">Titulo del Libro</p>
```

Comment:
Same, here I would define `book-title` instead, since you could have more than 1 type of title in your page further down the road.